test/passenger_test.rb

```diff
@@ -69,6 +69,18 @@
   end
 
   describe "net_expenditures" do
-    # You add tests for the net_expenditures method
+    it "properly calculate total passenger money charged" do
```

Comment:
For `net_expenditures` and `duration` you should test the edge case of a passenger that has no trips.